### PR TITLE
feature: add mobile option on ng new

### LINKED ISSUE #596: Add --mobile as option to "ng new"
This is a tracking issue for the work to support a mobile app flavor in the CLI.

The work to implement --mobile consists of 3 separate features, to be implemented in order:

* [ ] Boilerplate app manifest and inclusion in index.html
* [ ] Installation of service worker (stretch goal: only enabled for prod builds)
* [ ] App shell based initial application, with Universal support

For the first feature (boilerplate for the app manifest), the CLI should create a boilerplate file entitled `manifest.webapp`, with the following JSON structure (including values filled in from user prompts):

```json
{
  "short_name": "<prompt, default to Angular CLI App>",
  "name": "<prompt, default to Angular CLI App>",
  "icons": [
    {
      "src": "<prompt, default to 96x96 png of Angular logo>",
      "sizes": "96x96",
      "type": "image/png"
    }
  ],
  "start_url": "/index.html",
  "display": "standalone",
  "orientation": "portrait"
}
```

And in `index.html`, the following should be added to `<head>`:

```html
<link rel="manifest" href="/manifest.webapp">
```

Implementation details for the other 2 features are pending.

### PR DESCRIPTION
**Pull request by hansl**
Fixes #596.

### CODE REVIEW

File: addon/ng2/blueprints/ng2/files/__path__/manifest.webapp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "short_name": "<%= htmlComponentName %>",
```
Comment (alxhub) [inline addon/ng2/blueprints/ng2/files/__path__/manifest.webapp]: short_name should still be human readable. If CLI doesn't have something appropriate, just omit it.

**Comment by alxhub** [comment]
LGTM pending one comment.

File: addon/ng2/blueprints/ng2/index.js [hunk 2]
```diff
@@ -6,27 +7,47 @@ module.exports = {
...
     this.styleExt = 'css';
     this.version = require(path.resolve(__dirname, '..', '..', '..', '..', 'package.json')).version;
-    
+
     // Join with / not path.sep as reference to typings require forward slashes.
     const refToTypings = options.sourceDir.split(path.sep).map(() => '..').join('/');
+    const fullAppName = stringUtils.dasherize(options.entity.name)
+      .replace(/-(.)/g, (_, l) => ' ' + l.toUpperCase())
+      .replace(/^./, (l) => l.toUpperCase());
+console.log(options.mobile)
```
Comment (Brocco) [inline addon/ng2/blueprints/ng2/index.js]: Please remove the console.log

**Comment by Brocco** [comment]
One small nit, but otherwise lgtm

**Comment by hansl** [comment]
All done.

**Pull request merged**